src/components/App.js

```diff
@@ -1,12 +1,28 @@
 import React from "react";
 import SplashPage from "./SplashPage";
+import search from '../helpers.js'
 import "../css/App.css";
+
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      searchResults: []
```

Comment:
Thank you for adding this. It should help us pass that data to cards to populate info